v2/netmap/marshal.go

```diff
@@ -118,35 +131,35 @@ func (s *Selector) StableMarshal(buf []byte) ([]byte, error) {
 		err       error
 	)
 
-	n, err = proto.StringMarshal(nameSelectorField, buf[offset:], s.name)
+	n, err = protoutil.StringMarshal(nameSelectorField, buf[offset:], s.name)
 	if err != nil {
 		return nil, err
 	}
 
 	offset += n
 
-	n, err = proto.UInt32Marshal(countSelectorField, buf[offset:], s.count)
+	n, err = protoutil.UInt32Marshal(countSelectorField, buf[offset:], s.count)
 	if err != nil {
 		return nil, err
 	}
 
 	offset += n
 
-	n, err = proto.EnumMarshal(clauseSelectorField, buf[offset:], int32(s.clause))
+	n, err = protoutil.EnumMarshal(clauseSelectorField, buf[offset:], int32(s.clause))
 	if err != nil {
 		return nil, err
 	}
 
 	offset += n
 
-	n, err = proto.StringMarshal(attributeSelectorField, buf[offset:], s.attribute)
+	n, err = protoutil.StringMarshal(attributeSelectorField, buf[offset:], s.attribute)
 	if err != nil {
 		return nil, err
 	}
 
 	offset += n
 
-	n, err = proto.StringMarshal(filterSelectorField, buf[offset:], s.filter)
+	n, err = protoutil.StringMarshal(filterSelectorField, buf[offset:], s.filter)
```

Comment:
`n` can be omitted (`_, err = `)

Reply:
Please, don't repeat linter messages. `n` is outside this PR.

Reply:
oh, not seen GA/lint notifycation